test/extensions/filters/http/ext_authz/ext_authz_test.cc

```diff
@@ -329,6 +329,10 @@ TEST_F(HttpFilterTest, RequestDataWithPartialMessage) {
 
   data_.add("barfoo");
   EXPECT_EQ(Http::FilterDataStatus::StopIterationAndWatermark, filter_->decodeData(data_, true));
+
+  data_.add("more data after watermark is set is possible");
+  EXPECT_EQ(Http::FilterDataStatus::StopIterationAndWatermark, filter_->decodeData(data_, true));
```

Comment:
the first call should have `end_stream = false` parameter

Reply:
Oh, yeah. Thank you for this.

Reply:
modified in c6b9334.